src/Microsoft.TestPlatform.ObjectModel/Client/Payloads/ProxyAttachDebuggerToProcessAckPayload.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client
+{
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Class used to define the <see cref="EditorAttachDebuggerAckPayload"/> sent by the
+    /// vstest.console translation layers into design mode.
+    /// </summary>
+    public class EditorAttachDebuggerAckPayload
```

Comment:
>EditorAttachDebuggerAckPayload [](start = 17, length = 30)

Do update the file name to match the class name,

Reply:
Done.